middle_end/flambda2/to_cmm/to_cmm.ml

```diff
@@ -96,7 +96,7 @@ let unit0 ~offsets flambda_unit ~all_code =
   (* See comment in [To_cmm_set_of_closures] about binding [my_region] *)
   let env, _bound_var =
     Env.create_bound_parameter env
-      (Flambda_unit.toplevel_my_region flambda_unit)
+      (Flambda_unit.toplevel_my_region flambda_unit, [| Cmm.Int |])
```

Comment:
Could you introduce an alias for the machtype `[| Cmm.Int |]` in `to_cmm_shared.ml` (or `to_cmm_utils`, or `cmm_helpers.ml`), and use that alias here instead of building it by hand ?

As a general point: except for `to_cmm_shared.ml`, there should be no direct construction of `Cmm` values in the files of `to_cmm`.